uber/models.py

```diff
@@ -477,7 +477,7 @@ def guess_attendee_watchentry(self, attendee):
                                                          WatchList.email == attendee.email,
                                                          WatchList.birthdate == attendee.birthdate),
                                                      WatchList.last_name == attendee.last_name,
-                                                     WatchList.active == True)).all()
+                                                     WatchList.active == True))
```

Comment:
I recommend just returning ``.all()`` here.  ``self.query(...)`` returns a list-like object which responds to a lot of the same functions and operations as a list (you can iterate over it with ``for x in ...``, index it with ``...[0]``, etc), but it's not actually a ``list`` and so there can be some weird edge cases if someone expects to get a list back but actually gets back a queryset.

Sometimes we WANT to return the queryset, like if we want to return something that consumers can call ``.filter()`` on (``session.valid_attendees()`` is a good example of a case where we really do want to return a queryset for that very reason), but in this case we should probably just return a real ``list`` as given to us by ``.all()``.